tests/render/test_data_documentation_site_builder.py

```diff
@@ -92,6 +92,9 @@ def assert_how_to_buttons(
 
 @freeze_time("09/26/2019 13:42:41")
 @pytest.mark.rendered_output
+@pytest.mark.filterwarnings(
```

Comment:
These are GE-generated deprecation warnings - we filter them out in tests but they should be sent to the user.